scripts/smithy.lic

```diff
@@ -0,0 +1,137 @@
+=begin
+
+  Smithy service helper script
+
+  This script will request multiple services from the smithy. It tries to be
+  strict about what it will accept for input and to verify that a blacksmith
+  NPC is present before requesting services.
+
+  Usage:
+
+  ;smithy <count> <crit, damage, or sighting>
+
+  author: Kragdruk
+  version: 1.0
+  tags: util, ccf, duskruin, festival, merchant
```

Comment:
Do we have any thoughts on tags or some kind of taxonomy? I have been looking at existing tags in `;repo gui` to get some ideas but would be open to more consistency for these.

Reply:
Probably should have `wps` as a tag at the least